src/styles/login.css

```diff
@@ -0,0 +1,43 @@
+#card {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    margin: auto;
+    width: 35%;
+    height: 50%;
+    min-height: 600px;
+}
+
+#logo {
+    width: 100px;
+    height: 100px;
+    margin-top: 5%;
+}
+
+#card-content {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: center;
+    text-align: center;
+}
+
+#form-input-div {
+    padding-left: 20%;
+    padding-right: 20%;
+}
+
+.text-fields {
+    width: 100%;
+}
+
+#forgot {
+    float: left;
```

Comment:
Using the float property is general a kind of hacky thing to do. In this case, to get the text to be on the left side, you actually want to target the parent element `Typography` and set the `text-align` property appropriately.